### PR TITLE
Updates for purs v0.15

### PR DESCRIPTION
**Pull request by klntsky**

### CODE REVIEW

File: src/Jest.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
 // `void` is used here to suppress the Jest warning:
 // > A "describe" callback must not return a value.
-exports.describe = name => fn => () => describe(name, () => void fn());
+export const describe = name => fn => () => describe(name, () => void fn());
```
Comment (rnons) [inline src/Jest.js:3]: Thanks for your PR. Does this mean every user of purescript-jest has to set `NODE_OPTIONS=--experimental-vm-modules` as well?

Comment (klntsky) [reply src/Jest.js:3]: Yes, since PS compiler outputs ES modules starting from v0.15.

Comment (rnons) [reply src/Jest.js:3]: Cool, didn't know about it. Thanks.

**Comment by rnons** [comment]
Seems tests failed in gh-actions. I invited you as a collaborator, you can fix tests and do a release if you want.

**Pull request merged**